api/Document.json

```diff
@@ -4428,7 +4440,8 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "11.1"
+              "version_added": "11.1",
+              "notes": "Does not support the <code>navigator.permissions.query({name: 'storage-access'})</code>."
```

Comment:
Could you provide supporting evidence for this change?  (If this is accurate, I also think that it qualifies for `"partial_implementation": true`.)

Reply:
> Could you provide supporting evidence for this change?

<img width="853" alt="image" src="https://github.com/mdn/browser-compat-data/assets/10931297/f10c498e-50fe-488b-b1ad-326f0c17e92f">

> (If this is accurate, I also think that it qualifies for "partial_implementation": true.)

That's an interesting question!

Technically, the Permission API is a separate spec, but this particular policy of that has been integrated into [the Storage Access API](https://privacycg.github.io/storage-access/#permissions-integration). Safari has effectively implemented the older version of the spec but (yet?) upgraded to include this new part of the spec.

This does cause some [extra considerations that developers have to be mindful of](https://developer.mozilla.org/en-US/docs/Web/API/Storage_Access_API/Using#:~:text=because%20some%20browsers%20don%27t%20support%20the%20%22storage%2Daccess%22%20permission%2C%20which%20can%20cause%20the%20query()%20call%20to%20throw) (which is why I wanted to add this note). FYI, @chrisdavidmills is working on the docs and may expand upon this [in the current doc](https://developer.mozilla.org/en-US/docs/Web/API/Storage_Access_API#additions_to_other_apis) as well as add the relevant Permission API BCD to that doc.

However, I'm a little reluctant to mark this as a "partial implementation" since this is an extra requirement from the original spec. It was [added last year](https://github.com/privacycg/storage-access/pull/138) so is not super new, but is an extra requirement none-the-less.

I'm also conscious that I work for Chrome - another browser vendor - so while I want BCD to be accurate, I'd prefer to be cautious about regressing another vendors implementation unless this is super clear this is the right thing to do.

If we did add the `"partial_implementation": true` when does that apply from? From the beginning before this permission even existed? Only from when it was added to the spec (so they regressed from fully supported to partial supported?)? Do we have any precedent for that in BCD?

@johannhof / @chrisdavidmills do you have an opinion here? Should we reach out to Safari to see what they think (if so, who's best for that)?

Reply:
> Technically, the Permission API is a separate spec, but this particular policy of that has been integrated into [the Storage Access API](https://privacycg.github.io/storage-access/#permissions-integration). Safari has effectively implemented the older version of the spec but not (yet?) upgraded to include this new part of the spec. See [bug here](https://bugs.webkit.org/show_bug.cgi?id=249381).

Ah, if it's a case of the spec getting updated, then adding a subfeature is probably more appropriate.  I'm not sure what the name/description of the subfeature would be, though, hmm...

Reply:
Well it is the feature documented in `api.Permission.permission_storage-access` (also being updated in this PR) so it's a more a "related feature", which is already documented elsewhere in BCD.

I could just remove the note if you think that's best, but thought it was helpful as I got tripped up by this not being supported.

Reply:
FYI I just checked and another similar thing is this:

https://github.com/mdn/browser-compat-data/blob/eb349e5a3b586cb99539228de388c8c5f1f56d77/api/Document.json#L6162-L6165

Which shows up as this:

<img width="794" alt="image" src="https://github.com/mdn/browser-compat-data/assets/10931297/f803c409-1332-4311-96e1-2fa1faa6de2d">

So the API has "full support" but there's some notes about it.

On that Permissions API point I added the note to this API too.

Reply:
Reverted this change as per below so resolving.